### PR TITLE
Task/305 add consolev2 first screen test

### PR DESCRIPTION
**Pull request by AndyWhiteVega**
# Related issues 🔗

Closes #[Issue number here]

# Description ℹ️

Covers a lot of the Market Overlay form which appears at first load.
A PR is in to add more locators - for the overlay dialog and some column headings - once that is merged in we can revisit this for some light adjustments to keep the tests more robust.

# Demo 📺

Gif/video/images of new/corrected functionality if applicable

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: apps/trading-e2e/src/support/step_definitions/markets-page.step.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { forEach } from 'lodash';
```
Comment (jtsang586) [inline apps/trading-e2e/src/support/step_definitions/markets-page.step.ts:2]: This isn't used?

Comment (AndyWhiteVega) [reply apps/trading-e2e/src/support/step_definitions/markets-page.step.ts:2]: Yeah there is a couple of them just  removing them now :)

File: apps/trading-e2e/src/support/step_definitions/home-page.step.ts [hunk 2]
```diff
@@ -1,7 +1,126 @@
...
+    .then((mostRecentMarketInstrument) => {
+      cy.contains(mostRecentMarketInstrument.instrument.code).click();
+    });
+});
+
+When('I click the view full market list', () => {
+  cy.contains('Or view full market list').click();
+  cy.contains('Loading...').should('be.visible');
+  cy.contains('Loading...').should('not.exist');
+});
```
Comment (jtsang586) [inline apps/trading-e2e/src/support/step_definitions/home-page.step.ts:123]: I'm seeing quite a few type errors in this file and probably what's failing currently failing the build

**Review by jtsang586 — APPROVED** [review]
Lgtm

**Review by dexturr — APPROVED** [review]

File: apps/trading-e2e/src/support/step_definitions/home-page.step.ts (hunk 2 above)
Comment (AndyWhiteVega) [reply apps/trading-e2e/src/support/step_definitions/home-page.step.ts:145]: All typescript stuff now complete

**Pull request merged**